### PR TITLE
:construction: feat: new header implemented

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206246702422564

### CODE REVIEW

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: components/wallet/menu-mobile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  const handleOpenMenu = () => setIsOpen(true);
+
+  const handleCloseMenu = () => setIsOpen(false);
+
+  return (
+    <Box>
+      <Button
+        isIcon
+        gap="xs"
+        p="0.25rem"
```
Comment (MarioBatalha) [inline components/wallet/menu-mobile/index.tsx:22]: Put the value from the theme

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: components/wallet/index.tsx [hunk 2]
```diff
@@ -10,21 +10,28 @@ const Wallet: FC = () => {
...
+            >
+              <SuiNetwork />
+            </Box>
+          </>
         )}
         <AccountInfo />
       </Box>
       {!isConnected && (
-        <Box display={['none', 'none', 'none', 'flex']}>
+        <Box display={'flex'}>
```
Comment (git-marcopitra) [inline components/wallet/index.tsx:34]: remove `{}`

File: components/wallet/menu-mobile/main-menu.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  >
+    <Box zIndex="2" gridColumn="1/-1">
+      <Box
+        display={['flex', 'flex', 'flex', 'none']}
+        flexDirection="row-reverse"
+        pb="l"
+      >
+        <MenuButton handleClose={closeMenu} />
+      </Box>
+      <Typography m="xl" variant="title" size="small" color="onSurfaceVariant">
```
Comment (git-marcopitra) [inline components/wallet/menu-mobile/main-menu.tsx:27]: this color doesn't exist

File: views/pool-details/pool-form/component/receive-section/index.tsx [hunk 4]
```diff
@@ -11,7 +11,7 @@ const PoolReceiveSection: FC<ReceiveSectionProps> = ({ items }) => {
       <Typography variant="body" size="large" mb="1rem">
         You will receive
       </Typography>
-      <Box borderRadius="0.5rem" bg="surface" py="0.5rem">
+      <Box borderRadius="xs" bg="surface" py="0.5rem">
```
Comment (git-marcopitra) [inline views/pool-details/pool-form/component/receive-section/index.tsx:14]: py value can come from theme

File: views/pool-details/pool-form/component/receive-section/index.tsx [hunk 5]
```diff
@@ -11,7 +11,7 @@ const PoolReceiveSection: FC<ReceiveSectionProps> = ({ items }) => {
       <Typography variant="body" size="large" mb="1rem">
```
Comment (git-marcopitra) [inline views/pool-details/pool-form/component/receive-section/index.tsx:11]: mb value can come from theme

**Review by git-marcopitra — CHANGES_REQUESTED** [review]
u r not going over each review

**Review by MarioBatalha — APPROVED** [review]

File: components/wallet/menu-mobile/main-menu.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+import { FC } from 'react';
+
+import MenuButton from '../menu-button';
+import MenuSettingsList from '../menu-settings';
+import { MainMenuMobileProps } from './menu.types';
+import MobileMenuList from './menu-list';
+
+const MainMenu: FC<MainMenuMobileProps> = ({ closeMenu }) => (
+  <Box
+    m="0 1.25rem"
```
Comment (git-marcopitra) [inline components/wallet/menu-mobile/main-menu.tsx:11]: replace `m` to `mx`, and replace the value for theme based

File: components/wallet/menu-mobile/main-menu.tsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    display="flex"
+    minHeight="100%"
+    variant="container"
+    justifyItems="unset"
```
Comment (git-marcopitra) [inline components/wallet/menu-mobile/main-menu.tsx:15]: remove this prop

File: components/wallet/menu-settings/header.tsx [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+          scale: 1.05,
+          transition: { duration: 0.05, ease: easeInOut },
+        }}
+        borderRadius="50%"
+        alignItems="center"
+        justifyContent="center"
+        transition={{ duration: 0.5 }}
+      >
+        <CaretUpSVG
+          width="0.469rem"
```
Comment (git-marcopitra) [inline components/wallet/menu-settings/header.tsx:45]: replace `width` and `height` values to `100%`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Pull request merged**